package.json

```diff
@@ -201,6 +201,6 @@
     "typescript": "^5.0.4"
   },
   "engines": {
-    "node": "^14.13.1 || >=16.0.0"
+    "node": "^16.14.0 || >=18.0.0"
```

Comment:
[suggest] I agree with @Mouvedia's suggestion on <https://github.com/stylelint/stylelint/pull/6785#issuecomment-1565834412>. Node.js 16.13.0 is the first version of Node.js 16.x LTS series.

See https://nodejs.org/en/blog/release/v16.13.0

```suggestion
    "node": "^16.13.0 || >=18.0.0"
```

The [`ls-engines`](https://www.npmjs.com/package/ls-engines) check passes:

```console
$ npx ls-engines@latest
`node_modules` found; loading tree from disk...
┌────────┬─────────────────────────────────────────────────────────────────┐
│ engine │ Currently available latest release of each valid major version: │
├────────┼─────────────────────────────────────────────────────────────────┤
│ node   │ v20.3.1, v19.9.0, v18.16.1, v17.9.1, v16.20.1                   │
└────────┴─────────────────────────────────────────────────────────────────┘

┌───────────────────────────────────┬───────────────────────────┐
│ package engines:                  │ dependency graph engines: │
├───────────────────────────────────┼───────────────────────────┤
│ "engines": {                      │ "engines": {              │
│   "node": "^16.13.0 || >= 18.0.0" │   "node": ">= 16.13"      │
│ }                                 │ }                         │
└───────────────────────────────────┴───────────────────────────┘


Your “engines” field allows fewer node versions than your dependency graph does.
...
```

Surely, dev dependencies require higher Node.js versions. But we can ignore them because they're just for *dev*, and we should respect `engines.node` to support more users.

```console
$ npx ls-engines@latest --dev
...
Your “engines” field does not exactly match your dependency graph‘s requirements!
...
```